.github/workflows/wheels.yml

```diff
@@ -0,0 +1,34 @@
+name: Wheels
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build wheel ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "ubuntu-20.04", "windows-2019", "macos-11" ]
+    env:
+      CIBW_ARCHS_MACOS: x86_64 arm64 universal2
+      CIBW_TEST_COMMAND: pytest {project}/tests
+      CIBW_TEST_REQUIRES: pytest
```

Comment:
have you not specified the test-command and test-requires in the pyproject.toml file under `[tool.cibuildwheel]`?

Reply:
Yes, but for some reason it didn't cause cibuildwheel to run the tests.